03-transactions.md

```diff
@@ -397,25 +473,32 @@ committed HTLCs:
    in [Commitment Transaction](#commitment-transaction).
 1. Calculate which committed HTLCs need to be trimmed (see [Trimmed Outputs](#trimmed-outputs)).
 2. Calculate the base [commitment transaction fee](#fee-calculation).
-3. Subtract this base fee from the funder (either `to_local` or `to_remote`),
-   with a floor of 0 (see [Fee Payment](#fee-payment)).
-3. For every offered HTLC, if it is not trimmed, add an
+3. Subtract this base fee from the funder (either `to_local` or `to_remote`).
+If `option_anchor_outputs` applies to the commitment transaction,
```

Comment:
@rustyrussell why do you suggest to not add this block? The anchor values need to be subtracted from `to_local`/`to_remote` to construct a correct commitment tx.

Reply:
Small clarification (potentially related to rusty's question): in the case where there is only one anchor (because the remote output doesn't materialize), can you confirm that we will subtract only one time the 330 sats from the `to_local` output?

It's a bit painful to explicitly clarify in the steps, but it seems intuitive enough.